packages/dd-trace/test/appsec/iast/analyzers/sql-injection-analyzer.mysql.plugin.spec.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
-
+const fs = require('fs')
```

Comment:
Sorry, I didn't see this in the first review
```suggestion

const fs = require('fs')
```